lib/src/generic_dropdown_config_provider.dart

```diff
@@ -12,7 +12,7 @@ import 'package:flutter/widgets.dart';
 /// at the wrong position. With this config, the dropdown can
 /// utilize the root storybook as root screen key to calculate
 /// the position of the content.
-class GenericDropdownConfigProvider extends InheritedWidget {
+final class GenericDropdownConfigProvider extends InheritedWidget {
```

Comment:
it makes sense to use final here, it disallows subtyping outside of this library. so nobody can overwrite the provider and create breaking overwrites.